blocks/library/freeform/freeform-block.js

```diff
@@ -200,6 +250,25 @@ export default class FreeformBlock extends wp.element.Component {
 		}
 	}
 
+	onScroll() {
+		if ( this.toolbarElem ) {
+			let blockControls = findDOMNode( this.toolbarElem );
+			while ( blockControls ) {
+				if ( blockControls.classList.contains( 'editor-visual-editor__block-controls' ) ) {
+					break;
+				}
+				blockControls = blockControls.parentElement;
+			}
```

Comment:
I considered element.closest() but dismissed it because IE does not support it. I didn't know that the polyfill was available.